src/Routes/RoutesCommon.tsx

```diff
@@ -1,28 +1,107 @@
 import { getConfig } from 'config'
+import { lazy } from 'react'
 import { FaCreditCard, FaFlag } from 'react-icons/fa'
 import { RiExchangeFundsLine } from 'react-icons/ri'
+import { makeSuspsenseful } from 'utils/makeSuspenseful'
```

Comment:
👀

Reply:
Ok that's actually amazing not to forget to wrap lazy components in `<Suspense />` 🐐